storage/innobase/xtrabackup/doc/source/installation/apt_repo.rst

```diff
@@ -120,37 +92,37 @@ For more information about the pinning you can check the official
 .. _standalone_deb:
 
 Installing |Percona XtraBackup| using downloaded deb packages
-=============================================================
+================================================================================
 
 Download the packages of the desired series for your architecture from the
-`download page <https://www.percona.com/downloads/XtraBackup/>`_. Following
-example will download |Percona XtraBackup| 2.4.4 release package for *Debian*
+`download page <https://www.percona.com/downloads/XtraBackup/>`_. The following
+example will download |Percona XtraBackup| 8.0.12-1 release package for *Debian*
 8.0:
 
 .. code-block:: bash
 
-  $ wget https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.4/\
-  binary/debian/jessie/x86_64/percona-xtrabackup-24_2.4.4-1.jessie_amd64.deb
+  $ wget https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-8.0.12/\
+  binary/debian/jessie/x86_64/percona-xtrabackup-80_0.12-1.jessie_amd64.deb
```

Comment:
8.0.12 doesn't look like a correct version for me, please check the downloads page for correct links